angular-workspace/projects/ni/nimble-angular/rich-text/editor/nimble-rich-text-editor.directive.ts

```diff
@@ -60,6 +62,10 @@ export class NimbleRichTextEditorDirective {
         this.renderer.setProperty(this.elementRef.nativeElement, 'placeholder', value);
     }
 
+    public get validity(): RichTextValidity {
```

Comment:
Updates to directives should have corresponding updates to their spec files

Reply:
I have added `validity` and `checkValidity` default value test cases to the spec files.